cypress/integration/app_admin/app_admin.js

```diff
@@ -0,0 +1,45 @@
+import { Given, When, Then, Before, After } from 'cypress-cucumber-preprocessor/steps';
+
+const USER_NAME = 'test@unit.no';
+const APP_ADMIN_ROLE = 'App-admin';
+
+Before(() => {
+  cy.deleteUser(USER_NAME).then(() => {
+    cy.addUser(USER_NAME, APP_ADMIN_ROLE).then((userId) => {
+      cy.wrap(userId).as('userId');
+      cy.visit('/');
+      cy.wait(5000);
+
+      cy.get('[data-testid=create-author-button]').click();
+      cy.get('[data-testid=modal_next]').click();
+      cy.get('[data-testid=skip-connect-to-orcid]').click();
+
+      cy.setLanguage();
+      cy.visit('/');
+    });
+  });
+});
+
+Given('that the user is logged in', () => {
+  cy.get('@userId');
+});
+Given('they have the role of Application administrator', () => {});
+
+When('they look at any page in NVA', () => {
+  cy.visit('/');
+});
+
+Then('they see a menu containing', (tableData) => {
+  // | My Profile   |
+  // | Institutions |
+  // | Log Out      |
+  cy.get('[data-testid=menu]').click();
+  tableData.rawTable.forEach((row) => {

```

Comment:
dette var magisk! hvordan kommer `tableData` som argument, og hvordan vet man at det er en tabell så man kan bruke `rawTable`? 
får meg også til å tenke på typescript... har det vært vurdert?

Reply:
Tror PCB så på det og fant en del problemer. Men kan forsåvidt se på det på nytt. 

( må lese dokumentasjonen for å se at det er en tabell, veldig magisk, ja)

Reply:
kan jo lage en egen task på å undersøke, mulig det er en større jobb ja

Reply:
Mener det er en grunn til at vi ikke bruker typescript i frontend for testene, men husker ikke hva det var.

Reply:
Lager egen task på det